### DISCUSSION TITLE
How can I add styling/css to the captions

### DISCUSSION THREAD

**Discussion by mudit2108** [muxinc/media-chrome#1030] [Q&A] [answered]
I am not able to add css to the captions in the player. 
The only thing I am able to change it the background color by using this CSS property
`video::cue {
        background-color: rgba(0, 0, 0, 0.1);
}`
I have been trying to change the font size, color and add other CSS properties to cus but not able to do so

**Comment by luwes** [comment, marked as answer]
Sorry we missed this one. Explained here https://github.com/muxinc/media-chrome/discussions/720
